customize Magento admin logo

abrarpathan19 opened the pull request:
## Purpose of this pull request

<!-- REQUIRED Describe the goal and the type of changes this pull request covers. -->

This pull request (PR) customize Magento admin logo 

## Affected DevDocs pages

<!-- REQUIRED List the affected pages on devdocs.magento.com (URLs). Not needed for large numbers of files. -->

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-create.html#logo_declare
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/themes/theme-create.html#logo_declare
- https://devdocs.magento.com/guides/v2.1/frontend-dev-guide/themes/theme-create.html#logo_declare


## Links to Magento source code

<!--  OPTIONAL - REMOVE THIS SECTION IF NOT USED. If this pull request references a file in a Magento codebase repository, add it here. -->



<!-- 
If you are fixing a GitHub issue, note it using GitHub keyword format (https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository) to close the issue when this pull request is merged. Example: `Fixes #1234`

`master` is the default branch. Merged pull requests to `master` go live on the site automatically. Any requested changes to content on the `master` branch must be related to the released codebase. Any content related to future releases goes in the `develop` branch.

See Contribution guidelines (https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md) for more information.
-->

devops-devdocs commented:
An admin must run tests on this PR before it can be merged.

rogyar commented:
Also, please, make sure that your change is applicable only for 2.3. I believe it works for the previous versions as well. 
Thank you

abrarpathan19 commented:
Hi @rogyar
Yes, I agreed this topic is more suitable for your mentioned link, I am also want to add this content on this page but in admin logo section last line redirects to other page that is why I have added that content over there.

Can I move this content on your suggested link?

rogyar commented:
Hi @abrarpathan19. Sure, you can move it there

abrarpathan19 commented:
Hi @rogyar I have updated suggested changes, please review 

Thanks

rogyar commented:
Hi @abrarpathan19. Please, change the "Affected DevDocs" section in the PR description correspondingly. Also, I see changes only for versions 2.1 and 2.3, what about 2.2? 

Thank you.

abrarpathan19 commented:
Hello @rogyar it is reflected in all 2.1, 2.2, 2 3

abrarpathan19 commented:
I have applied changes in 2.1 file it will be effected in 2.2 as well

abrarpathan19 commented:
@rogyar Thanks to approve my PR

jeff-matthews commented on guides/v2.1/frontend-dev-guide/themes/admin_theme_create.md:
```diff
@@ -74,6 +74,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
 To override it, in your theme directory, create a `web/images` sub-directory, and add your custom file named `magento-logo.svg`. 
 If you want to use the file with other name and/or format, you need to additionally declare it as described in [Declaring theme logo]({{ page.baseurl }}/frontend-dev-guide/themes/theme-create.html#logo_declare).
 
+To update Admin theme logo, take the following steps:
```
```suggestion
To customize the Admin theme logo:
```

jeff-matthews commented on guides/v2.1/frontend-dev-guide/themes/admin_theme_create.md:
```diff
@@ -74,6 +74,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
 
+To update Admin theme logo, take the following steps:
+
+1. After you create admin theme then You can customize Magento admin logo.
```
This is not a step. It is a statement. Remove it.

jeff-matthews replied:
@abrarpathan19, this is the third time I'm asking you to make the same revision. It is not an efficient use of time.

jeff-matthews replied:
The requested revision is to delete line 79.

jeff-matthews commented on guides/v2.1/frontend-dev-guide/themes/admin_theme_create.md:
```diff
@@ -74,6 +74,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
+
+1. After you create admin theme then You can customize Magento admin logo.
+
+2. Create a new `xml` in theme named `app/design/adminhtml/<Vendor>/<theme>/Magento_Backend/layout/admin_login.xml`. 
```
```suggestion
2. Create a new XML file in the theme named `app/design/adminhtml/<Vendor>/<theme>/Magento_Backend/layout/admin_login.xml`. 
```

jeff-matthews commented on guides/v2.1/frontend-dev-guide/themes/admin_theme_create.md:
```diff
@@ -74,6 +74,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
+
+2. Create a new `xml` in theme named `app/design/adminhtml/<Vendor>/<theme>/Magento_Backend/layout/admin_login.xml`. 
+
+ Use following example in `admin_login.xml`.
```
```suggestion
1. Add the following sample code to the new file.
```

jeff-matthews commented on guides/v2.1/frontend-dev-guide/themes/admin_theme_create.md:
````diff
@@ -74,6 +74,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
+2. Create a new `xml` in theme named `app/design/adminhtml/<Vendor>/<theme>/Magento_Backend/layout/admin_login.xml`. 
+
+ Use following example in `admin_login.xml`.
+ ```xml
````
This code block needs to be indented 4 spaces.

jeff-matthews commented on guides/v2.1/frontend-dev-guide/themes/admin_theme_create.md:
````diff
@@ -74,6 +74,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
+    </body>
+</page>
+```
+3. Place your custom logo in following directory 
````
```suggestion
3. Add your custom logo to the `app/design/adminhtml/<Vendor>/<theme>/web/images/` directory. 
```

jeff-matthews commented on guides/v2.1/frontend-dev-guide/themes/admin_theme_create.md:
````diff
@@ -74,6 +74,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
+</page>
+```
+3. Place your custom logo in following directory 
+ `app/design/adminhtml/<Vendor>/<theme>/web/images/custom-logo.svg`
````
Delete this line

jeff-matthews commented on guides/v2.3/frontend-dev-guide/themes/admin_theme_create.md:
```diff
@@ -71,6 +71,27 @@ In the default `Magento/backend` theme `lib/web/images/magento-logo.svg` is used
 To override it, in your theme directory, create a `web/images` sub-directory, and add your custom file named `magento-logo.svg`. 
 If you want to use the file with other name and/or format, you need to additionally declare it as described in [Declaring theme logo]({{ page.baseurl }}/frontend-dev-guide/themes/theme-create.html#logo_declare).
 
+To update Admin theme logo, take the following steps:
```
Apply all updates from the 2.1 file to this file

abrarpathan19 commented:
Hi @jeff-matthews, I have applied suggestions, please review

jeff-matthews commented:
@abrarpathan19, you have not applied ALL of the suggested edits. I will not merge this Pr until you do so.

abrarpathan19 commented:
Hi @jeff-matthews I have updated all files, please review again.
Thanks :)

abrarpathan19 commented:
Hi, @jeff-matthews I am sorry but I have applied all the suggested changes. Now old content is not available "To update Admin theme logo, take the following steps:" 
please review:  http://tinyurl.com/yxk4go3m

abrarpathan19 commented:
Hi, @jeff-matthews I have removed line 79, But the concern is that I have added that line because it is necessary to create admin theme to change admin logo. 
Thanks

jeff-matthews commented:
The introductory sentence implies that the theme has already been created since you cannot customize something that does not exist.

jeff-matthews commented:
running tests

abrarpathan19 commented:
@jeff-matthews thank  you
